### DISCUSSION TITLE
allowed hosts error

### DISCUSSION THREAD

**Discussion by gookenhaim** [saleor/saleor#6011] [Q&A]
Hi! 

In mirumee/saleor's settings.py, we can see that there's no allowed_hosts variable. 
And whenever I tried to runserver in django, the debug mode threw an allowed_hosts error saying that I should add my ip to the variable even when it wasn't provided by Saleor's settings.py 

So I just made that up in the file but it didn't work as well.
What should I do to solve this? I really want to use Saleor... thanks

**Comment by korolevpavel** [comment]
Hi @gookenhaim 
What ip have you added?

**Comment by IKarbowiak** [comment]
`ALLOWED_HOSTS` variable exists in `saleor/settings.py`:
https://github.com/mirumee/saleor/blob/master/saleor/settings.py#L392.
You should add your ip there.
